CHANGELOG.md

```diff
@@ -20,25 +20,191 @@ We’ve updated the header component to support account information and links. A
+
+For precompiled stylesheets, note the following path changes:
+
+- Copy or serve `node_modules/dist/nhsuk/nhsuk-frontend.min.css`, not the previous `node_modules/dist/nhsuk.min.css` stylesheet
+- Extract `nhsuk-frontend-<VERSION-NUMBER>.min.css` from the GitHub release zip file, not the previous `css/nhsuk-<VERSION-NUMBER>.min.css` stylesheet
+
+##### If you’re using JavaScript
+
+For JavaScript imported using a bundler, consolidate all `import` or `require()` calls to `nhsuk-frontend/packages/components/*` into a single statement:
+
+Before:
+
+```mjs
+import initButtons from 'nhsuk-frontend/packages/components/button/button.js'
+import initCheckboxes from 'nhsuk-frontend/packages/components/checkboxes/checkboxes.js'
+
+// Initialise all button components
+initButtons();
+
+// Initialise all radios components
+initRadios();
+```
+
+After:
+
+```mjs
+import { initButtons, initCheckboxes } from 'nhsuk-frontend'
+
+// Initialise all button components
+initButtons();
+
+// Initialise all radios components
+initRadios();
+```
+
+Or alternatively, you can initialise individual components only:
+
+```js
+import { Button, Checkboxes } from 'nhsuk-frontend';
+
+const $button = document.querySelector('.app-button')
+const $checkboxes = document.querySelector('.app-checkboxes')
+
+// Initialise single button component
+new Button($button);
+
+// Initialise single checkboxes component
+new Checkboxes($checkboxes);
+```
+
+For precompiled JavaScript, note the following path changes:
+
+- Copy or serve `node_modules/dist/nhsuk/nhsuk-frontend.min.js`, not the previous `node_modules/dist/nhsuk.min.js` script
+- Extract `nhsuk-frontend-<VERSION-NUMBER>.min.js` from the GitHub release zip file, not the previous `js/nhsuk-<VERSION-NUMBER>.min.js` script
+
+Then include the script before the closing `</body>` tag of your page using the `type="module"` attribute, and run the `initAll` function to initialise all the components.
+
+Before:
+
+```html
+  <!-- // ... -->
+  <script src="/javascripts/nhsuk-frontend.min.js" defer></script>
+</head>
+```
+
+After:
+
+```html
+  <!-- // ... -->
+  <script type="module" src="/javascripts/nhsuk-frontend.min.js"></script>
+  <script type="module">
+    import { initAll } from '/javascripts/nhsuk-frontend.min.js'
+    initAll()
+  </script>
+</body>
+```
+
+##### If you’re using Nunjucks
+
+1. Change the list of paths in `nunjucks.configure()` to search within `node_modules/nhsuk-frontend/dist`:
+
+Before:
+
+```mjs
+nunjucks.configure([
+  'node_modules/nhsuk-frontend/packages/components',
+  'node_modules/nhsuk-frontend/packages/macros'
+])
+```
+
+After:
+
+```mjs
+nunjucks.configure([
+  'node_modules/nhsuk-frontend/dist/nhsuk/components',
+  'node_modules/nhsuk-frontend/dist/nhsuk/macros',
+  'node_modules/nhsuk-frontend/dist/nhsuk',
+  'node_modules/nhsuk-frontend/dist'
+])
+```
+
+##### If you’re copying or serving assets
+
+Replace `packages/` with `dist/nhsuk` when copying or serving NHS.UK frontend assets:
```

Comment:
Is it worth mention that this section also applies to serving static images like logos and favicons? Nearly all services should be doing that.